docs_versioned_docs/version-ros2jazzy/components/yaml/attachments/a300/top_plate_pacs.mdx

```diff
@@ -0,0 +1,30 @@
+<table>
+<tr>
+<td>
+<center>
+<figure>
+    <img src={require("./img/top_plate_pacs.png").default} width="250" />
+</figure>
+</center>
+</td>
+<td>
+
+```yaml
+platform:
+  attachments:
+    - name: top_plate
+      type: a300.top_plate
+      model: pacs
+      parent: default_mount
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+      enabled: true
+```
+</td>
+</tr>
+</table>
+
+Attach accessories to the top plate mounts by setting the accessory's `parent` parameter to one of the grid mounting
+locations displayed above. The grid mounting locations span from `${name}_mount_a1` to `${name}_mount_e9`, where the
+front left-most location is the `a1` mount and the rear right-most location is the `e9` mount. The letters correspond
+to the columns and the number to the rows.
```

Comment:
Add a blank line to the end of MDX files.

Reply:
This file already has a newline at the end of the file